src/components/Navbar/SearchInput.tsx

```
@@ -0,0 +1,18 @@
+import { SearchC } from "@/src/styles/Navbar.styled";
+import React from "react";
+import { FiSearch } from "react-icons/fi";
+type SearchInputProps = {
+  // user:
+};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)